start-a-local-cluster.md

```diff
@@ -31,37 +31,35 @@ Data is stored on-disk and client/server communication is secure. Setup involves
     # To exit: CTRL + D.
     ```
 
-4. [Run some queries](/basic-sql-commands.html).
+3. [Run some queries](/basic-sql-commands.html).
+4. Check out the Admin UI by pointing your browser to `http://<your local host>:26257`. You can find your local host by running `hostname` in your shell.    
```

Comment:
I'm pretty sure `http://localhost:26257` will work. Ditto for the `standard mode` link